projects/packages/masterbar/src/admin-color-schemes/colors/classic-bright/colors.scss

```diff
@@ -1,3 +1,52 @@
 @import "variables";
 @import "../admin";
 @import "sidebar-notice";
+
+// Specific overrides for this color scheme.
+.admin-color-classic-bright {
```

Comment:
We're going to remove Calypso version of the single-site view (nav unification) as part of the Untangling Calypso project in the future. Given that, is it that worth it to maintain this kind of override (for Classic Bright and Powder Snow)?

I'd imagine this could cause some confusion for future devs 🤔 

For the record, I'm not sure for this set of colors, which comes first, Calypso's or wp-admin's. 😅